aikau/src/main/resources/alfresco/services/actions/CopyMoveService.js

```diff
@@ -80,7 +80,17 @@ define(["dojo/_base/declare",
       moveAPI: "slingshot/doclib/action/move-to/node/",
 
       /**
-       * Sets up the service using the configuration provided. 
+       * URL to call to create a link to a document
+       *
+       * @instance
+       * @type {string}
+       * @default
+       */
```

Comment:
When adding a new attribute or function to a module we add a "since" annotation so that the JSDoc shows when the capability was added, so here for example you'd want to add:

```
@since 1.0.91
```